File: JuanGonzalez_GenerarTiempos.cpp
#include <iostream>
#include <ctime>

using namespace std;

//declaro funciones
int fibonacci(int N);
float get_time(int N);
int main(){
	int i;
	//for de impresion de primeros 35 terminos
	for(i =0;i<35;i++){
		//imprimo i y el tiempo de fibo(i)
		cout<<i<<","<<get_time(i)<<endl;
	}
	return 0;
}
int fibonacci(int N){
	//casos base
	if(N==0){
		return 0;
	}if(N==1){
		return 1;
	}
	//recursividad
	else{
		return fibonacci(N-1)+fibonacci(N-2);
	}
}
float get_time(int N){
	clock_t t;
	//tiempo inicial
	t=clock();
	fibonacci(N);
	//tiempo despues de correr fibo(N)
	t=clock()-t;
	//lo convierto en float
	float time=((float)t)/CLOCKS_PER_SEC;
	return time;
}

File: JuanGonzalez_GenerarTiempos.py
import numpy as np
import matplotlib.pyplot as plt
import time

def fibonacci(N):
	if N==0:#caso base
		return 0
	elif N==1:#caso base
		return 1
	else:#caso recursivo
		return fibonacci(N-1)+fibonacci(N-2)
def get_time(N):
	t0=time.time()#tiempo inicial
	fibonacci(N)#corro fibo(N)
	return time.time()-t0#retorno la diferencia de tiempos
for i in range(0,35):#recorrido para imprimir de 0 a 34 fibo(n)
	print str(i)+","+str(get_time(i))
